globus_automate_client/cli/helpers.py

```diff
@@ -51,6 +51,10 @@ def process_input(
     Turn input strings into dicts per input format type (InputFormat)
     """
     input_dict = None
+
+    if input_arg is None:
```

Comment:
Since this has potential to be `None`, can we update the type annotation to reflect that? Also, do we need to create the `input_dict` at the top of the func?

Reply:
Good points, just pushed the changes